package.json

```diff
@@ -16,7 +16,8 @@
     "@prisma/client": "^5.7.1",
     "next": "14.0.4",
     "react": "^18",
-    "react-dom": "^18"
+    "react-dom": "^18",
+    "react-icons": "^4.12.0"
```

Comment:
fyi you'll probably run into a conflict here as I'm already using on another branch off main